packages/renderer-protocol/inverseRpc/services/friendRequestService.ts

```diff
@@ -48,27 +37,19 @@ export function registerFriendRequestKernelService(port: RpcServerPort<RendererP
         // Handle send friend request
         const sendFriendRequest = await requestFriendship(req)
 
-        let sendFriendRequestReply: SendFriendRequestReply = {}
-
-        // Check the response type
-        if (sendFriendRequest.reply?.friendRequest) {
```

Comment:
In the new version the `sendFriendRequest.reply` is checked instead of the `sendFriendRequest.reply?.friendRequest`, is this semantically the same?

Reply:
No, it's not the same. But the older one was unnecessarily checking the friendRequest because if we already have the reply, it means we already have the friendRequest as well

Reply:
Great! 👍🏻